library/item/item_extension/src/main/java/org/quiltmc/qsl/item/extension/mixin/crossbow/CrossbowItemAccessor.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.item.extension.mixin.crossbow;
+
+import net.minecraft.item.CrossbowItem;
+import net.minecraft.item.ItemStack;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+@Mixin(CrossbowItem.class)
+public interface CrossbowItemAccessor {
+	@Invoker("getSpeed")
+	static float quilt$getSpeed(ItemStack stack) {
+		throw new IllegalStateException("Mixin injection failed.");
```

Comment:
i probably should have dug in first instead of saying "oh, just follow the standards!"
```suggestion
		throw new IllegalStateException("Invoker injection failed.");
```